packages/eslint-plugin-wpcalypso/lib/rules/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const LOCALIZABLE_URLS = [
+	// apps.wordpress.com/**
+	/^(https?:)?\/\/apps\.wordpress\.com/i,
+
+	// automattic.com/cookies/
+	/^(https?:)?\/\/automattic\.com\/cookies\/?$/i,
+
+	// automattic.com/privacy/
+	/^(https?:)?\/\/automattic\.com\/privacy\/?$/i,
+
+	// wordpress.com/tos/
+	/^(https?:)?\/\/wordpress\.com\/tos\/?$/i,
+
+	// wordpress.com/blog/
+	/^(https?:)?\/\/wordpress\.com\/blog\/?$/i,
```

Comment:
Should we support particular blog links? I'm not sure if that makes sense though, since I assume each language of the blog has its own articles. Worth double-checking of course.

Reply:
Blog articles URLs are non-localizable. Similarly to the forums, we only want to localize the home url of the blog site.